var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -154,6 +158,19 @@ class Bazel(Package):
     patch('disabledepcheck.patch', when='@0.3.2:+nodepfail')
     patch('disabledepcheck_old.patch', when='@0.3.0:0.3.1+nodepfail')
 
+    # patches for compiling bazel-4.1: with gcc-11
+    # these are derived from
+    # https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/29084/
+    patch('gcc11_1.patch', when='@4.1:%gcc@11:')
+    patch('gcc11_2.patch', when='@4.1:%gcc@11:')
+    patch('gcc11_3.patch', when='@4.1:%gcc@11:')
+    patch('gcc11_4.patch', when='@4.1:%gcc@11:')
+
+    # bazel-4.0.0 does not compile with gcc-11
```

Comment:
What about bazel 3? Should this be `@:3`?

Reply:
I need to look at bazel 3 and see what needs to be done.